msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -174,15 +215,26 @@ public async Task<AuthenticationResult> AcquireTokenSilentAsync(IEnumerable<stri
         }
 
         /// <summary>
-        /// Attempts to acquire the access token from cache. Access token is considered a match if it AT LEAST contains all the requested scopes.
-        /// This means that an access token with more scopes than requested could be returned as well. If access token is expired or 
-        /// close to expiration (within 5 minute window), then refresh token (if available) is used to acquire a new access token by making a network call.
+        /// Attempts to acquire and access token for the <paramref name="account"/> from the user token cache, with advanced parametersking a network call.
         /// </summary>
-        /// <param name="scopes">Array of scopes requested for resource</param>
-        /// <param name="account">Account for which the token is requested <see cref="Account"/></param>
-        /// <param name="authority">Specific authority for which the token is requested. Passing a different value than configured does not change the configured value</param>
-        /// <param name="forceRefresh">If TRUE, API will ignore the access token in the cache and attempt to acquire new access token using the refresh token if available</param>
-        /// <returns></returns>
+        /// <param name="scopes">scopes requested to access a protected API</param>
+        /// <param name="account">Account for which the token is requested. <see cref="IAccount"/></param>
+        /// <param name="authority">Specific authority for which the token is requested. Passing a different value than configured in the application constructor
+        /// narrows down the selection of tenants for which to get a tenant, but does not change the configured value</param>
+        /// <param name="forceRefresh">If <c>true</c>, the will ignore the access token in the cache and attempt to acquire new access token 
```

Comment:
If <c>true</c>, ignore any access token in the cache and attempt to acquire a new one